tests/unit/environments/MutiLabelTestEnvironment.h

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
The name of this PR is add binary Label fixture, but in order to prove the GaussianCheckerboard can works for mutilabelclass, I add this global fixture and apply MulticlassOCAS_unittest to use it. Hope it won't confuse other people. And I will rename this pr after the rest of it is good to go :)